cms/static/js/index.js

```diff
@@ -141,20 +141,26 @@ define(["domReady", "jquery", "underscore", "js/utils/cancel_on_escape", "js/vie
             e.preventDefault();
             $('.courses-tab').toggleClass('active', tab === 'courses');
             $('.libraries-tab').toggleClass('active', tab === 'libraries');
+            $('.programs-tab').toggleClass('active', tab === 'programs');
+
             // Also toggle this course-related notice shown below the course tab, if it is present:
-            $('.wrapper-creationrights').toggleClass('is-hidden', tab === 'libraries');
+            $('.wrapper-creationrights').toggleClass('is-hidden', tab === 'libraries' || 'programs');
```

Comment:
I don't think this test does what it is intended to do. I think it should be:

    tab === 'libraries' || tab === 'programs'

I believe that ``'libraries' || 'programs'`` just evaluates to 'libraries'.